README.md

```diff
@@ -23,21 +23,73 @@
 
 ## Introduction
 
-Jujutsu is a
-[Git-compatible](https://martinvonz.github.io/jj/latest/git-compatibility)
-[DVCS](https://en.wikipedia.org/wiki/Distributed_version_control). It combines
-features from Git (data model,
-[speed](https://github.com/martinvonz/jj/discussions/49)), Mercurial (anonymous
-branching, simple CLI [free from "the
-index"](https://martinvonz.github.io/jj/latest/git-comparison#the-index),
-[revsets](https://martinvonz.github.io/jj/latest/revsets), powerful
-history-rewriting), and Pijul/Darcs ([first-class
-conflicts](https://martinvonz.github.io/jj/latest/conflicts)), with features not
-found in most of them
-([working-copy-as-a-commit](https://martinvonz.github.io/jj/latest/working-copy),
-[undo functionality](https://martinvonz.github.io/jj/latest/operation-log),
-automatic rebase, [safe replication via `rsync`, Dropbox, or distributed file
-system](https://martinvonz.github.io/jj/latest/technical/concurrency)).
+Jujutsu is a powerful [version control system](https://en.wikipedia.org/wiki/Version_control)
+system for software projects. You use it to get a copy of the code, track
+changes to the code, and finally publish those changes for others to see and
+use. Jujutsu is unlike most other systems, because internally it abstracts
+the data model used to store your content from its user facing algorithms and
+interface. This allows it to serve as a version control interface with many
+possible physical backends, such as Git (today), while also open to other
+potential backends, like Mercurial, or even more centralized systems like
+Perforce.
```

Comment:
I see, I was wondering about other things mostly as a comparison. Do you think there are any other comparable examples of VCS backends that could be plugged in? It would be nice to have at least 2.

Reply:
Breezy handles Git already and is snapshot based, so it would be a candidate for a backend.

Reply:
There's also the cloud-based backend we use at Google. That's quite different from most backends (simply by being cloud-based) that it might be interesting to the reader to hear about.